packages/docs-app/src/styles/_examples.scss

```diff
@@ -533,6 +533,23 @@
   }
 }
 
+#{example("Section")} {
+  .docs-example {
+    align-items: start;
+  }
+
+  .metadata-panel {
```

Comment:
@CPerinet for future reference: styling of example content should be done in CSS like this